### DISCUSSION TITLE
ERROR: Large endpoint Cookie, removing endpointID => Apple TV

### DISCUSSION THREAD

**Discussion by cchervitz** [maxileith/homebridge-appletv-enhanced#377] [Q&A]
Anybody else see this error throughout their logs? I think it's mostly innocuous, but I can't figure out what's causing it or what it really means. Thanks for any suggestions!

**Comment by maxileith** [comment]
I need additional context to dig into this problem. If you still experience these errors I would suggest you open a bug report in the issues section.
